src/main/java/com/linkedin/kmf/services/configs/TopicManagementServiceConfig.java

```diff
@@ -76,7 +77,7 @@
               ConfigDef.Importance.LOW, REBALANCE_INTERVAL_MS_DOC)
       .define(PARTITIONS_TO_BROKER_RATIO_CONFIG,
               ConfigDef.Type.DOUBLE,
-              2.0,
+              PARTITIONS_TO_BROKER_RATIO_DEFAULT,
```

Comment:
I see how this constant is being used now.  So I think the correct usage pattern of the various XConfig objects is to instantiate them and then let them do all the error checking and population of default values.  That means the constant does not need to be public which means it's safe to remove it.  Probably we still need to make all the defaults constants, but that is a patch for different time.